packages/contracts-bedrock/src/dispute/FaultDisputeGameN.sol

```diff
@@ -517,7 +517,12 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
             value_ = starting.raw();
         } else if (_ident == LocalPreimageKey.DISPUTED_OUTPUT_ROOT) {
             // Load the disputed proposal's output root
-            Claim disputed = getClaim(disputedRoot.raw(), disputedPos, _daItem);
+            Claim disputed;
+            if (disputedPos.raw() == 1) {
```

Comment:
I would suggest adding a comment here

Reply:
> I would suggest adding a comment here

Fixed here

Reply:
Fixed in https://github.com/ethstorage/optimism/pull/41/commits/74242dce1d3fa38ca34c3629e0094a1ee292727a#r1683759252